RELEASE_V1_GOALS.md

```diff
@@ -0,0 +1,24 @@
+
+
+We, the active developers of the community fork, have agreed on these tentative goals for the next release of this project:
```

Comment:
nit, use more # to style headers

# Development strategy/Release goals/...

#### (An might skip the 'an'?) HTM accessible in C++, Python

#### (The) NetworkAPI

#### Ongoing Research

### Code organization